demos/common/python/openvino/model_zoo/model_api/adapters/remote_adapter.py

```diff
@@ -0,0 +1,86 @@
+"""
+ Copyright (c) 2021 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+try:
+    import ovmsclient
+    ovmsclient_absent = False
+except ImportError:
+    ovmsclient_absent = True
+
+from .model_adapter import ModelAdapter, Metadata
+
+
+class RemoteAdapter(ModelAdapter):
```

Comment:
Yes, it is still a draft version, not for a review. Not sure if I should delete it or not. It will be overwritten by the final version of adapter by OVMS team in other PR anyway, but it might be useful to see the overall structure of the remote_adapter.py file